### DISCUSSION TITLE
SSDs for 24/7 home server?

### DISCUSSION THREAD

**Discussion by manoloxxl** [R3NE07/Futro-S740#19] [Q&A]
Hi,

Are the linked cheep SSDs also good options when the system is planed to run as a 24/7 home server (Home assist or iobroker) or will it be a better option to buy a WD Red or similar?

best Regards

**Comment by CaneTLOTW** [comment]
I use WD RED

**Comment by GeStenZ** [comment]
You have to watch your SMART data. There are reports on cheap consumer SSDs failing after a couple of months. Server SSDs with a high TBW rating should last longer.
